spatialprofilingtoolbox/cggnn/scripts/upload_importance.py

```diff
@@ -0,0 +1,49 @@
+"""Uploads importance score output from a cg-gnn instance to the local db."""
+
+from argparse import ArgumentParser
+
+from pandas import read_csv
+
+from spatialprofilingtoolbox.db.database_connection import DatabaseConnectionMaker
+from spatialprofilingtoolbox.db.importance_score_transcriber import transcribe_importance
+
+
+def parse_arguments():
+    """Process command line arguments."""
+    parser = ArgumentParser(
+        prog='spt cggnn upload_importance',
+        description='Upload importance score output from a cg-gnn model to db.'
+    )
+    parser.add_argument(
+        '--filename',
+        type=str,
+        help='',
+        required=True
+    )
+    parser.add_argument(
+        '--target_var',
```

Comment:
I guess it's the cohort stratification. It's meant to be whatever target variable the GNN was trained to classify for. I suppose it should be contained in the output artifact, but I didn't design the GNN pipeline to behave like that (yet), and it would make the output no longer a CSV but something more complicated (a JSON or a zipped folder containing text and a CSV)?

Reply:
Hm well why don't we just add a call to `spt cggnn upload-importance`, with a string variable providing this argument value, to the end of the training? Then the "user" doesn't need to worry about it.

Reply:
I don't follow. What is upload-importance calling that will provide the stratification variable name?

Reply:
I'm saying, we can add:
```sh
spt cggnn upload-importance --target-var="$TARGET_VAR"
```
to whatever script does training, and make sure that `TARGET_VAR` environment variable is set correctly by the caller script. I guess I'm suggesting that there does not need to be a separate manually run feature export step like this, it can be automatic.

Reply:
Just like the `fractions_transcriber` does its stuff right after views refreshing.

Reply:
So, I guess here in [`run_sql.py`](https://github.com/nadeemlab/SPT/blob/3e8c2a6f2afc05c41044878e4b63481f0c165094/spatialprofilingtoolbox/cggnn/scripts/run_sql.py#L148).
Could skip the CLI step and just directly call `transcribe_importance`.